plugins/devtools-backend/api-report.md

```diff
@@ -11,15 +11,17 @@ import { DiscoveryService } from '@backstage/backend-plugin-api';
 import express from 'express';
 import { ExternalDependency } from '@backstage/plugin-devtools-common';
 import { HttpAuthService } from '@backstage/backend-plugin-api';
-import { Logger } from 'winston';
+import { LifecycleService } from '@backstage/backend-plugin-api';
+import { LoggerService } from '@backstage/backend-plugin-api';
 import { PermissionsService } from '@backstage/backend-plugin-api';
+import { SignalService } from '@backstage/plugin-signals-node';
 
 // @public (undocumented)
 export function createRouter(options: RouterOptions): Promise<express.Router>;
 
 // @public (undocumented)
 export class DevToolsBackendApi {
-  constructor(logger: Logger, config: Config);
+  constructor(logger: LoggerService, config: Config);
```

Comment:
@drodil, any chance you can reply to the above, please?

Reply:
So this will not be a breaking change as winston implements the logger service interface. The logger service is the new backend replacement for logger.

Reply:
Thanks, enjoy the holiday!